### PR TITLE
DesktopStreamer OSX: zip the application for Puppet deployment

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

**Comment by dnachbaur** [comment]
0.6 was released, but it says git master in Changelog.md.

Bump version to 0.7 or 0.6.1?

**Comment by rdumusc** [comment]
Version 0.6 has been released, but until there are new features just keep updating the patch version (currently it is at 0.6.1). I didn't update to 0.6.2 for this PR because from a user perspecitve it really doesn't change anything, it's just a nice-to-have functionality for developers...

**Comment by dnachbaur** [comment]
Ah, didn't see it was already 0.6.1.

Still, from the Changelog.md I don't see a 'Deflect 0.6'.

**Comment by rdumusc** [comment]
Yeah, changelog says git master (0.6), probably should have been changed to Deflect 0.6 at release time. Will do it now.

File: doc/RelNotes.md [hunk 1]
```diff
@@ -20,13 +20,14 @@ if you find any other issue with this release.
...
 
-## git master (0.6)
+## Deflect 0.6
 
 * Many classes of the Server part have been renamed
 * Various API fixes
 * Project-wide codestyle cleanup
 * DesktopStreamer: Minor fixes for desktop interaction, only show checkbox on
   supported platforms (OSX)
+* DesktopStreamer OSX: zip the application for Puppet deployment
```
Comment (dnachbaur) [inline doc/RelNotes.md:9]: this is part of the (unreleased) 0.6.1. Maybe put in under a new section, e.g. git-master like in https://github.com/Eyescale/Equalizer/pull/463/files#diff-f04a81819f7614362e00f5804d9ceb0a?

Besides, you want to add the release dates as well to the changelog?

File: CMakeLists.txt [hunk 2]
```diff
@@ -15,8 +15,7 @@ set(VERSION_MINOR "6")
 set(VERSION_PATCH "1")
 set(VERSION_ABI 1)
 
-set(DEFLECT_DESCRIPTION
-  "The client library to write applications for DisplayCluster")
+set(DEFLECT_DESCRIPTION "The C++ library to stream pixels to DisplayCluster")
```
Comment (dnachbaur) [inline CMakeLists.txt]: It's actually not bound to DisplayCluster anymore. You can stream and receive pixels just with Deflect.

Comment (rdumusc) [inline CMakeLists.txt]: Yes...  what about: "A fast C++ library for streaming pixels and events"?

Comment (dnachbaur) [inline CMakeLists.txt]: Yes! +1

**Pull request merged**